engine/apps/user_management/utils.py

```diff
@@ -0,0 +1,35 @@
+import typing
+
+from apps.user_management.constants import (
+    AlertGroupTableColumn,
+    AlertGroupTableColumns,
+    AlertGroupTableColumnTypeChoices,
+    AlertGroupTableDefaultColumnChoices,
+)
+
+if typing.TYPE_CHECKING:
+    from apps.user_management.models import User
+
+
+def default_columns() -> typing.List[AlertGroupTableColumn]:
+    columns = [
+        {"name": column.label, "id": column.value, "type": AlertGroupTableColumnTypeChoices.DEFAULT.value}
+        for column in AlertGroupTableDefaultColumnChoices
+    ]
+    return columns
```

Comment:
maybe this function would better fit in `apps/user_management/constants.py` along with `AlertGroupTableDefaultColumnChoices`?